src/Model/List/ListDecorator.php

```diff
@@ -113,15 +110,18 @@ public function getIterator(): Traversable
 
     public function exists(): bool
     {
-        return $this->list->exists();
+        if (method_exists($this->list, 'exists')) {
+            return call_user_func([$this->list, 'exists']);
+        }
+        return false;
```

Comment:
```suggestion
        return $this->count() > 0;
```
A list "exists" if it has values.

Reply:
Updated